src/main/java/com/synopsys/integration/alert/provider/DefaultEmailHandler.java

```diff
@@ -113,10 +113,10 @@ public FieldAccessor updateFieldAccessor(final MessageContentGroup contentGroup,
                                         .map(AggregateMessageContent::getSubTopic)
                                         .anyMatch(Optional::isPresent);
         if (!hasSubTopic) {
-            final Boolean filterByProject = fieldAccessor.getBoolean(CommonDistributionConfiguration.KEY_FILTER_BY_PROJECT).orElse(false);
+            final Boolean filterByProject = fieldAccessor.getBoolean(BlackDuckDistributionUIConfig.KEY_FILTER_BY_PROJECT).orElse(false);
```

Comment:
See my comment above this isn't a BlackDuck specific field.  This is being used in the DefaultEmailHandler which isn't provider specific.  The constants don't belong in BlackDuckDistributionUIConfig